pages/docs/concepts/asyncapi-document/adding-messages.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Adding Messages
+weight: 140
+---
+
+Adding [messages](../message) in an AsyncAPI document primarily involves defining channels, operations, and messages. This process is crucial for documenting the exchange of data between your applications. In AsyncAPI, you define the interaction of your application with a message broker in terms of channels and operations. [Channels](../channel) are the medium through which messages are sent or received, while operations represent the actions of publishing or subscribing to these channels.
+
+Here is a diagram explaining messages:
+
+```mermaid
+graph TD;
+    A[AsyncAPI Message] --> B[Headers]
+    A --> C[Payload]
+    B --> D[Header Content]
+    C --> E[Payload Content]
```

Comment:
```suggestion
    A[AsyncAPI Message] --> B[Headers]
    A --> C[Payload]
```

I think this one could be shorter. Unless there's a difference between Headers/Payload and Header/Payload Content.